pyramid_multiauth/__init__.py

```diff
@@ -37,10 +37,11 @@ def track_policy(event):
             print("We selected policy %s" % event.policy)
 
     """
-    def __init__(self, policy, request):
+    def __init__(self, policy, request, userid):
```

Comment:
We can indeed (I will), although nobody is supposed to instantiate those events :)